index.html

```diff
@@ -0,0 +1,2 @@
+<LINK REL=StyleSheet HREF="style.css" TYPE="text/css" MEDIA=screen>
+<h1>Hello Microverse </h1>
```

Comment:
- [ ] Kindly note that your HTML file doesn't follow the proper convention. use the below suggestion while making the required changes.
```suggestion
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Hello Microverse</title>
  <link rel="stylesheet" href="style.css">
</head>
<body>
  <h1>Hello Microverse </h1>
</body>
</html>
```